hal/src/boron/pinmap_defines.h

```diff
@@ -170,41 +170,37 @@
 
 #if PLATFORM_ID == PLATFORM_ESOMX
 
-#define TOTAL_PINS 45
+#define TOTAL_PINS 39
 #define TOTAL_ANALOG_PINS 8
 #define FIRST_ANALOG_PIN A0
 
 // Digital pins
 #define D0 0
 #define D1 1
 #define D2 2
-#define D3 3
-#define D4 4
-#define D5 5
-#define D6 6
-#define D7 7
-#define D8 8
-#define D9 9
-#define D10 10
-#define D11 11
-#define D12 12
-#define D13 13
-#define D14 14
-#define D15 15
-#define D16 16
-#define D17 17
-#define D18 18
-#define D19 19
-#define D20 20
-#define D21 21
-#define D22 22
-#define D23 23
-#define D24 24
-#define D25 25
-#define D26 26
-#define D27 27
-#define D28 28
-#define D29 29
+#define D5 3
+#define D8 4
+#define D9 5
+#define D10 6
+#define D11 7
+#define D12 8
+#define D13 9
+#define D14 10
+#define D15 11
+#define D16 12
+#define D17 13
+#define D18 14
+#define D19 15
+#define D20 16
+#define D21 17
+#define D22 18
+#define D23 19
+#define D24 20
```

Comment:
This doesn't look right
```
/* D21 / B3      - 17 */ { NRF_PORT_1,    13,          PIN_MODE_NONE, PF_NONE, ADC_CHANNEL_NONE, 2,                 3,                8, EXTI_CHANNEL_NONE, 0},
/* D24 / C0      - 18 */ { NRF_PORT_1,    8,           PIN_MODE_NONE, PF_NONE, ADC_CHANNEL_NONE, PWM_INSTANCE_NONE, PWM_CHANNEL_NONE, 8, EXTI_CHANNEL_NONE, 0},
```

Reply:
D22/D23 is still missing in the pinmap_hal.c